libs/templates/src/lib/who-we-are/sections/our-values/index.tsx

```diff
@@ -58,7 +58,7 @@ export const WhoWeAreValues = () => {
       cardSliderProps={{
         cards: ourValuesCards,
         variant: 'ContentBottom',
-        slideClasses: 'max-w-[242px] md:max-w-[296px]',
+        slideClasses: 'max-w-[286px] md:max-w-[296px]',
```

Comment:
Change to 286px because it is using 286px for mobile and tablet views in Figma. It cause different design compare to Figma. 
<img src="https://github.com/deriv-com/deriv-com-v2/assets/142988136/1ed4dec3-d2c2-46de-a11d-aec21fb3fe95" width="300px" /><img src="https://github.com/deriv-com/deriv-com-v2/assets/142988136/d0240780-13d8-4ba1-b810-8ad1fb1ce5cc" width="230px" />